cmd/dep/status.go

```diff
@@ -273,6 +274,36 @@ func (cmd *statusCommand) Run(ctx *dep.Ctx, args []string) error {
 	return nil
 }
 
+func (cmd *statusCommand) runOld(ctx *dep.Ctx, args []string, p *dep.Project, sm gps.SourceManager, params gps.SolveParameters) error {
+	solver, err := gps.Prepare(params, sm)
```

Comment:
Since we want to check update for all the projects, set `params.ChangeAll = true` before preparing a solver.